gongt.go

```diff
@@ -377,7 +381,15 @@ func (n *NGT) Open() *NGT {
 			return n
 		}
 	case Cosine:
-		// TODO: not exists C API
+		if C.ngt_set_property_distance_type_cosine(prop, ebuf) == ErrorCode {
+			n.errs = append(n.errs, newGoError(ebuf))
+			return n
+		}
+	case NormalizedAngle:
+		// TODO: not exists in C API
+		return n
```

Comment:
I think append `not implement error` before retun is better

Reply:
Thank you!
I implemented returning error.